sfge/src/main/java/com/salesforce/rules/fls/apex/operations/ValidationConverter.java

```diff
@@ -246,6 +268,15 @@ private void convertSoqlQueryInfo(
                 continue;
             }
 
+            if (isUserModeAccessLevel) {
```

Comment:
On the one hand, this is consistent with the convention established by the other checks, and it should work just fine.
On the other, passing a boolean parameter all the way down the stack like this just so you can `continue` in a loop feels inherently weird.
We'll keep what you've got for now, but I'll add an item to the spreadsheet to revisit it later.

Reply:
Actually, I have a another idea! Since a new validation converter is created for every vertex, I moved the boolean to be an instance variable. Let me know what you think.